doc/man7/provider-storemgmt.pod

```diff
@@ -105,6 +105,15 @@ further loading.
 
 OSSL_FUNC_store_close() frees the provider side context I<ctx>.
 
+When a provider-native object is created by a storemgmt it would be unsuitable
```

Comment:
I think it would be better to spell it out as _store manager_ rather than _storemgmt_.  Not vitial.